### PR TITLE
Remove redundant local variables

### PR DESCRIPTION
**Pull request by sigee**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java [hunk 1]
```diff
@@ -231,126 +231,126 @@ private static int zzUnpackAttribute(String packed, int offset, int [] result) {
...
+		//zzStartRead = zzEndRead = s.offset;
+		zzStartRead = s.offset;
+		zzEndRead = zzStartRead + s.count - 1;
+		zzCurrentPos = zzMarkedPos = s.offset;
+		zzLexicalState = YYINITIAL;
+		zzReader = reader;
+		zzAtEOF  = false;
+	}
+
+
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java]: All the line endings above were changed?

File: src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java [hunk 2]
```diff
@@ -481,13 +481,8 @@ public void yypushback(int number)  {
...
     int zzEndReadL = zzEndRead;
     char [] zzBufferL = zzBuffer;
-    char [] zzCMapL = ZZ_CMAP;
-
-    int [] zzTransL = ZZ_TRANS;
-    int [] zzRowMapL = ZZ_ROWMAP;
-    int [] zzAttrL = ZZ_ATTRIBUTE;
 
-    while (true) {
+      while (true) {
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java]: This whitespace change should be reverted? (In the github interface at least this makes the while aligned with code it contains.)

File: src/org/zaproxy/zap/extension/ascan/ChallengeCallbackAPI.java [hunk 3]
```diff
@@ -102,19 +102,17 @@ public void cleanExpiredCallbacks() {
...
      */
     public String getCallbackUrl(String challenge) {
-        String callbackUrl = "http://" 
-                + Model.getSingleton().getOptionsParam().getProxyParam().getProxyIp() + ":" 
-                + Model.getSingleton().getOptionsParam().getProxyParam().getProxyPort() + "/" 
-                + getPrefix() + "/" 
-                + challenge;
-        
-        /* Key is not currently used for shorcuts... very interesting
+        /* Key is not currently used for shortcuts... very interesting
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascan/ChallengeCallbackAPI.java]: This commented code could be removed as well.

File: src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java (hunk 2 above)
Comment (thc202) [reply src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java]: I'd suggest reverting all changes to this file, they will be lost once we regenerate it.

Comment (sigee) [reply src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java]: @kingthorin, @thc202, all changes reverted in this file.

File: src/org/zaproxy/zap/extension/ascan/ChallengeCallbackAPI.java (hunk 3 above)
Comment (sigee) [reply src/org/zaproxy/zap/extension/ascan/ChallengeCallbackAPI.java]: @thc202, the comment is removed as well.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**